samples/sampler/package.json

```diff
@@ -24,18 +24,20 @@
     "@enact/i18n": "^4.9.0-alpha.3",
     "@enact/sandstone": "../../",
     "@enact/spotlight": "^4.9.0-alpha.3",
-    "@enact/storybook-utils": "^5.1.4",
+    "@enact/storybook-utils": "enactjs/storybook-utils#feature/storybook8",
```

Comment:
After Merging https://github.com/enactjs/storybook-utils/pull/71, would you please update the branch to `feature/enact_v5`?
Also, please update the version of storybook modules to 8.1.6.